src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. In this case, you should remove the tag name from `.stars__star` and simply use a class for styling.